pkgs/by-name/sl/sleek-grub-theme/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, withBanner ? "Grub Bootloader" # use override to specify your own banner text
+, style ? "white" # use override to specify one of "dark" / "orange" / "bigSur"
```

Comment:
I think the build fails if `style` is not valid.

Is it good to add an assertion to make sure `style` is valid at eval time?

Reply:
Good idea!  added assert now.